model/tps/spans.yaml

```diff
@@ -0,0 +1,32 @@
+groups:
+  - id: span.tps.http.server
+    type: span
+    stability: development
+    brief: 'This document defines semantic conventions for a HTTP server spans of Transaction Processing Systems.'
+    extends: span.http.server
+    span_kind: server
+    attributes:
+      - ref: tps.system.name
+        requirement_level: required
+      - ref: tps.transaction.id
+        requirement_level: required
+      - ref: tps.transaction.owner.id
+      - ref: tps.task.id
+      - ref: tps.program.name
+      - ref: tps.unit_of_work.id
+
+  - id: span.tps.rpc.server
+    type: span
+    stability: development
+    brief: 'This document defines semantic conventions for Remote Procedure Call (RPC) server spans of Transaction Processing Systems'
+    extends: span.rpc.server
```

Comment:
FYI: we're going to actively update and revise this span definition within RCP semconv stabilization - there will be some breaking changes.